src/core/Microsoft.PowerToys.Settings.UI/ViewModels/PowerRenameViewModel.cs

```diff
@@ -2,14 +2,136 @@
 // The Microsoft Corporation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
+using System.Runtime.CompilerServices;
 using Microsoft.PowerToys.Settings.UI.Helpers;
+using Microsoft.PowerToys.Settings.UI.Lib;
+using Microsoft.PowerToys.Settings.UI.Views;
 
 namespace Microsoft.PowerToys.Settings.UI.ViewModels
 {
     public class PowerRenameViewModel : Observable
     {
+        private const string ModuleName = "PowerRename";
+
+        private PowerRenameSettings Settings { get; set; }
+
         public PowerRenameViewModel()
         {
+            Settings = SettingsUtils.GetSettings<PowerRenameSettings>(ModuleName);
+            _powerRenameEnabled = Settings.properties.MruEnabled.value;
```

Comment:
Same as before, there is no need to keep track of local copies of `Settings.properties...`